src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -702,6 +711,20 @@ private static StringBuilder getSessionDaysReport(ListingData listingData)
         return sb;
     }
 
+    private static StringBuilder getEccReport(ListingData listingData)
+            throws JsonProcessingException {
+        if (!(listingData instanceof EccReportData)) {
+            throw new IllegalStateException((LISTING_DATA_STATE_EXCEPTION + "EccReportData"));
+        }
+        var reportData = (EccReportData) listingData;
+        var sb = new StringBuilder();
+        if (CollectionUtils.isNotEmpty(reportData.getReportDetails())) {
+            sb.append(REPORT_OFFICE).append(reportData.getReportDetails().get(0).getOffice()).append(NEW_LINE);
```

Comment:
Don't you want report office to be written irrespective of whether or not there are any details?
Why don't you just have a property office in EccReportData rather than trying to get it from a collection that might not exist?

Reply:
yes, refactored it. office is now the property of reportData